README.md

```diff
@@ -1,47 +1,55 @@
 Maliit
+======
 
 Maliit provides a flexible and cross-platform input method framework. It has a
 plugin-based client-server architecture where applications act as clients and
 communicate with the Maliit server via input context plugins. The communication
-link currently uses D-Bus. Maliit is an open source framework (LGPL 2) with
-open source plugins (BSD).
+link currently uses D-Bus. Maliit is an open source framework (LGPL 2.1) with
+an production-quality keyboard plugin (LGPL 3.0).
 
 This is the Maliit framework. For a virtual keyboard, see
-https://github.com/maliit/keyboard .
+https://github.com/maliit/keyboard.
```

Comment:
Since this is markdown now, I guess this should be a markdown link. Or perhaps we can drop this whole sentence, and just make the `a production-quality keyboard plugin` a link to keyboard instead?